api/types/constants.go

```diff
@@ -273,6 +273,9 @@ const (
 	// KindClusterAlert is a resource that conveys a cluster-level alert message.
 	KindClusterAlert = "cluster_alert"
 
+	// KindPolicy is an RBAC policy resource.
+	KindPolicy = "policy"
```

Comment:
`policy` is an extremely generic name, maybe `access_policy` would be a bit better?. I could see other types of policies coming up, unless the plan is to serve all future "policies" with this resource kind as well.